libraries/picosystem.cpp

```diff
@@ -20,11 +20,11 @@ namespace picosystem {
 
   #ifndef DYNAMIC_BUFFER
   #ifdef PIXEL_DOUBLE
-    color_t _fb[120 * 120];
+    color_t _fb[120 * 120] = { };
```

Comment:
An empty initializer initializes to `0`'s.  No initializer leaves it randomized.